library/src/scala/NamedTuple.scala

```diff
@@ -19,7 +19,12 @@ object NamedTuple:
 
   def unapply[N <: Tuple, V <: Tuple](x: NamedTuple[N, V]): Some[V] = Some(x)
 
+  @annotation.compileTimeOnly("NamedTupleBuilder.Builder should have been eliminated by calling the build method.")
+  final class NamedTupleBuilder[N <: Tuple]():
```

Comment:
It looks technically overkill to introduce a class for this and the require it be compileTimeOnly. Why not just the simple `build` method?
```scala
  def build[N <: Tuple]()[V <: Tuple](x: V): NamedTuple[N, V] = x
```

Reply:
Ah I see, I had forgotten it was still experimental. But I think we can probably merge this before or at the same time as NamedTuples. Since NamedTuple is already @experimental it would be no additional hurdle if it also imported clauseInterleaving,

Reply:
added your suggestion